classes/Product.php

```diff
@@ -7062,18 +7062,20 @@ public function setWsPositionInCategory($position)
             'ORDER BY `position`'
         );
 
-        if ($position > count($result)) {
-            WebserviceRequest::getInstance()->setError(
-                500,
-                $this->trans(
-                    'You cannot set a position greater than the total number of products in the category, starting at 1.',
-                    [],
-                    'Admin.Catalog.Notification'
-                ),
-                135
-            );
+        if (count($result)!= 0) {
```

Comment:
```suggestion
        if (count($result)!= 0) {
        if (count($result) != 0) {
```

PHP CS Fixer needs this